samples/Rewrite/FileParser/RewriteReaderException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Rewrite.FileParser
+{
+    public class RewriteReaderException : Exception
```

Comment:
Do we need a custom Exception type? Does any one of the existing .NET exceptions make sense here instead?